app-emulation/fs-uae-launcher/fs_uae_launcher-2.8.3.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="FS-UAE Launcher - a GUI for FS-UAE"
+DESCRIPTION="FS-UAE is a cross-platform Amiga emulator based on updated \
+emulation code from WinUAE. FS-UAE uses SDL for input, OpenAL for audio \
+and OpenGL for graphics.
+
+This package contains the launcher, a graphical user interface for \
+setting up FS-UAE."
+HOMEPAGE="https://fs-uae.net/launcher"
+COPYRIGHT="2011-2017 Frode Solheim"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://fs-uae.net/stable/${portVersion}/fs-uae-launcher-${portVersion}.tar.gz"
+CHECKSUM_SHA256="8ce764031013516daab76e996eff9e9fdead6c0f00b4e497fc927f7719d8a233"
+SOURCE_DIR="fs-uae-launcher-$portVersion"
+PATCHES="fs_uae_launcher-$portVersion.patchset"
+ADDITIONAL_FILES="Misc_BoingballTick"
+
+ARCHITECTURES="?x86 !x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	fs_uae_launcher$secondaryArchSuffix = $portVersion
+	cmd:fs_uae_launcher$secondaryArchSuffix
+	"
+
+REQUIRES="
+    haiku$secondaryArchSuffix
+    pyqt_python3$secondaryArchSuffix
+    pip_python3$secondaryArchSuffix
+    fs_uae$secondaryArchSuffix
+    "
+
+BUILD_REQUIRES="
+    haiku${secondaryArchSuffix}_devel
+    "
+
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:msgfmt
+	cmd:python3
+	"
+
+BUILD()
+{
+	make
+}
+
+INSTALL()
+{
+	make install
+
+	mkdir -p $appsDir/fs-uae-launcher
+	cp -a usr/local/share/fs-uae-launcher/* $appsDir/fs-uae-launcher
+
+	chmod +x $appsDir/fs-uae-launcher/fs-uae-launcher
+	addattr -f $portDir/additional-files/Misc_BoingballTick -c VICN BEOS:ICON $appsDir/fs-uae-launcher/fs-uae-launcher
```

Comment:
Wrap before 80th character

Reply:
Why addatr?!
Check the other ports how they does it!

Reply:
@miqlas Because it is not a binary. You can't apply an rdef to a Python script. I did check the other ports and then confirmed on IRC the correct way to do it.